include/zephyr/pm/state.h

```diff
@@ -407,6 +428,21 @@ static inline const struct pm_state_info *pm_state_get(uint8_t cpu,
 	return NULL;
 }
 
+static inline const char *pm_state_to_str(enum pm_state state)
+{
+	ARG_UNUSED(state);
+
+	return NULL;
+}
+
+static inline int pm_state_from_str(const char *name, enum pm_state *out)
+{
+	ARG_UNUSED(name);
+	ARG_UNUSED(out);
+
+	return 0;
+}
```

Comment:
These two helpers don't need mocks IMO, they should only be used if PM is used in the first place, so just leave them out of CONFIG_PM is not defined